keras/applications/mobilenetv2.py

```diff
@@ -454,73 +448,39 @@ def MobileNetV2(input_shape=None,
 
 def _inverted_res_block(inputs, expansion, stride, alpha, filters, block_id):
     in_channels = inputs._keras_shape[-1]
-    prefix = 'features.' + str(block_id) + '.conv.'
     pointwise_conv_filters = int(filters * alpha)
     pointwise_filters = _make_divisible(pointwise_conv_filters, 8)
-    # Expand
-
-    x = Conv2D(expansion * in_channels, kernel_size=1, padding='same',
-               use_bias=False, activation=None,
-               name='mobl%d_conv_expand' % block_id)(inputs)
-    x = BatchNormalization(epsilon=1e-3, momentum=0.999,
-                           name='bn%d_conv_bn_expand' %
-                           block_id)(x)
-    x = Activation(relu6, name='conv_%d_relu' % block_id)(x)
+    x = inputs
+    prefix = 'mobl_expanded_conv_{}_'.format(block_id)
```

Comment:
It seems better to change it more concisely. How about `'block_{}_'.format(block_id)`?